bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -204,6 +216,7 @@ public void run() {
         }
         mPendingWriteTasks.add(runnable);
         mExecutorService.execute(runnable);
+
```

Comment:
Feel free to remove this new extra line (unless for organizational purposes).

Reply:
I'll double check, thanks.

Reply:
I removed this line but also used this as an opportunity to replace `mExecutorService.execute(runnable)` with `doInBackground(runnable)`.